server/src/server/index.ts

```diff
@@ -7,30 +7,46 @@ import passport from 'passport'
 import * as path from 'path'
 import { MODELS_DIR, ROUTES_DIR } from '../var/config'
 import { globFiles } from '../helpers'
+import MongoStore from 'connect-mongo'
 import router from '../router'
 import cors from 'cors'
 
 const app: express.Express = express()
 
+const ENVIRONMENT = process.env.ENVIRONMENT
 for (const model of globFiles(MODELS_DIR)) {
   require(path.resolve(model))
 }
 
 app.set('views', path.join(__dirname, '../../src/views'))
 app.set('view engine', 'pug')
 
-const sessionMiddleware = session({
+const sessionConfig: session.SessionOptions = {
   saveUninitialized: true,
   resave: true,
   proxy: true,
-  secret: 'no_secret', //leaving this like this for now
+  secret: process.env.SECRET || 'superdupersecret', //leaving this like this for now
```

Comment:
```suggestion
  secret: process.env.SECRET || 'superdupersecret',
```
Now that this is fine...